Tests/Twig/DataTableExtensionTest.php

```diff
@@ -6,14 +6,16 @@
 use CrossKnowledge\DataTableDundle\Tests\UsesContainerTrait;
 use Symfony\Component\DependencyInjection\Container;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use PHPUnit\Framework\TestCase;
+use Twig\Extension\AbstractExtension;
 
 require_once __DIR__.'/../UsesContainerTrait.php';
 
-class DataTableExtensionTest extends \PHPUnit_Framework_TestCase
+class DataTableExtensionTest extends TestCase
 {
     use UsesContainerTrait;
 
-    protected function getFunctionByName(\Twig_Extension $extension, $name)
+    protected function getFunctionByName(AbstractExtension $extension, $name)
```

Comment:
```suggestion
    /**
     * Return function of extension by name
     *
     * @param AbstratExtension $extension
     * @param string           $name
     */
    protected function getFunctionByName(AbstractExtension $extension, string $name)
```
Please add documentation

Reply:
Done.